### ISSUE TITLE
40 Winks

### ISSUE THREAD

**Issue by Dartomic** [socram8888/tonyhax#123]
**tonyhax version**:
- Latest cd iso

**Installation method**:
- It's the cd iso. I pop it in after Unirom boots up from the memory card

**Console model**:
- SCPH 5501 and SCPH 101

**Integrity check**: If it boots, does the built-in integrity check succeed?
- I'm not sure what this is.

**BIOS version**: If you can get to boot, which version of the BIOS does it report?
- v4.5 on PSOne. 
- v3.0 on SCPH 5501

**Target game**:
- 40 Winks. I don't know the game code.

**Bug explanation**:

I have made two copies of this game. Every other game I have burned works. Most of them work great on these CD's that I'm using. But the TonyHax CD only prints this to the screen when I try to boot 40 Winks:

Initializing CD
Disc error type D code 12 (x11)
Init failed
Reinitializing kernel
Copying patches to 000009EC
Installing modchip patch
Installing FreePSXBoot patch
Swap CD now

**Comment by socram8888** [comment]
So what you are doing is booting Unirom from the memory card using FreePSXBoot, then booting into tonyhax, and then into 40 Winks?

**Comment by Dartomic** [comment]
Yeah.

I've had issues with games not working well with Unirom's version of FreePSXboot. The memory card I was able to transfer files from on my computer is too slow for FreePSXboot, but not TonyHax. So I used TonyHax on that memory card, to boot into the Unirom cd and install FreePSXboot from there, onto the faster card that I couldn't transfer files onto completely from my Arduino due to pin 3's power requirement.
